packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -136,32 +140,37 @@ function renderHeader(header, content, onSelect, onToggle) {
 	];
 
 	if (content) {
+		const caretText = expanded
+			? t('COLLAPSIBLE_PANEL_COLLAPSE', { defaultValue: 'Collapse panel' })
+			: t('COLLAPSIBLE_PANEL_EXPAND', { defaultValue: 'Expand panel' });
+
 		const defaultCaret = (
 			<Button
 				className={classNames(css.toggle, 'toggle')}
 				bsStyle="link"
 				key={2}
 				onClick={onToggle}
+				title={caretText}
 			>
 				<Icon key={header.length} name="talend-caret-down" />
 			</Button>
 		);
 		wrappedHeader.push(defaultCaret);
 	}
-	return wrappedHeader;
+	return <div className={classNames(css['panel-heading'], 'panel-heading')}>{wrappedHeader}</div>;
 }
 
 function getKeyValueContent(content) {
 	return (
-		<dl>
-			{content.map((item, index) => (
-				<div key={index} className={css.content}>
-					<dt className={css.label}>
-						<Label>{item.label}</Label>
-					</dt>
-					<dd className={css.description}>{item.description}</dd>
-				</div>
-			))}
+		<dl className={css.content}>
+			{content.map((item, index) => [
+				<dt className={css.label} key={`${index}_label`}>
+					<Label>{item.label}</Label>
+				</dt>,
+				<dd className={css.description} key={`${index}_desc`}>
+					{item.description}
```

Comment:
i would add an empty string if the description is undefined, so label don't get misaligned with the wrong description